docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,20 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Mirroring and rewriting the docker.io registry
+
+The default Docker registry `docker.io` is handled in a special way, because
+every push and pull operation gets internally suffixed with `/library` if no
```

Comment:
Can we s/suffixed/normalized/? In case of `docker.io/foo` -> `docker.io/library/foo`, the "suffix" only relates to the namespace.